src/server/templates/typescript.ts

```diff
@@ -345,8 +351,8 @@ export interface Database {
                     )
                     if (relation) {
                       return `{
-                        ${relation.columns
-                          .sort(({ name: a }, { name: b }) => a.localeCompare(b))
+                        ${schemaColumns
```

Comment:
Just a sanity check, before you were passing different columns (views.columns, relation.columns) - it's correct that it's `schemaColumns` in all cases now as you're fetching them separately, right?

Reply:
Yup, in this case we need to fetch it from `schemaColumns`